pkg/operator/certrotation/signer.go

```diff
@@ -153,7 +159,7 @@ func needNewSigningCertKeyPair(annotations map[string]string, refresh time.Durat
 	validity := notAfter.Sub(notBefore)
 	at80Percent := notAfter.Add(-validity / 5)
 	if time.Now().After(at80Percent) {
-		return true, fmt.Sprintf("past its latest possible time %v", at80Percent)
+		return true, fmt.Sprintf("past refresh time (80%% of validity): %v", at80Percent)
```

Comment:
nit: do you think it will help if we also include the validity in the message?

Reply:
Yes, this would show happy path, where certs are updated at 80%, as the cert can be force refreshed in the future